libtraceevent/parse-filter.c

```diff
@@ -1624,9 +1624,11 @@ int pevent_filter_clear_trivial(struct event_filter *filter,
 		case FILTER_TRIVIAL_FALSE:
 			if (filter_type->filter->boolean.value)
 				continue;
+			/* fall through */
 		case FILTER_TRIVIAL_TRUE:
 			if (!filter_type->filter->boolean.value)
 				continue;
+			/* fall through */
```

Comment:
Wait, I'm not sure what the code does.  Shouldn't it be "break" instead of fall-through?